.ci/docker/common/install_onnx.sh

```diff
@@ -31,7 +32,7 @@ pip_install \
 pip_install coloredlogs packaging
 
 pip_install onnxruntime==1.18.1
-pip_install onnx==1.17.0
+pip install -i https://test.pypi.org/simple/ onnx==1.18.0rc1
```

Comment:
Maybe use no-deps

Reply:
Why is that? The original line does not use --no-deps